### PR TITLE
Replace global.registry with global.imageRegistry

### PR DESCRIPTION
**Pull request by yardenshoham**
Bitnami has the most popular helm charts and they named the global image registry `global.imageRegistry` so I suggest to stick to that standard.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by yardenshoham** [comment]
@sreekarjami

File: helm/opendistro-es/Chart.yaml [hunk 1]
```diff
@@ -26,4 +26,4 @@ name: opendistro-es
 sources:
 - https://pages.git.viasat.com/ATG/charts
 # Chart version
-version: 1.13.1
+version: 2.0.0
```
Comment (sreekarjami) [inline helm/opendistro-es/Chart.yaml:29]: The version here represents the current ODFE version. Please use 1.13.1

Comment (yardenshoham) [reply helm/opendistro-es/Chart.yaml]: Ok

**Comment by sreekarjami** [comment]
Hi @yardenshoham, 
Thanks for the change. Please post some test results, that the change is not impacting the helm and we should be good to merge the PR. 

Thanks for the contribution.

**Comment by yardenshoham** [comment]
I can attest I successfully deployed this helm chart after the change.

**Review by peterzhuamazon — APPROVED** [review]

**Review by gaiksaya — APPROVED** [review]

**Pull request merged**